nupic.core.kdev4

```diff
@@ -0,0 +1,4 @@
+[Project]
+CreatedFrom=CMakeLists.txt
+Manager=KDevCMakeManager
```

Comment:
rm this file, please

Reply:
probably you had the file first check to git, and later added to gitignore - it still stays in the repo. Just let the file removed here, and keep your local version - now it should not show in the public git, but you should keep your local config.